traffic_ops/traffic_ops_golang/routing/routes.go

```diff
@@ -445,7 +445,7 @@ func Routes(d ServerData) ([]Route, []RawRoute, http.Handler, error) {
 		{1.1, http.MethodGet, `federations/?(\.json)?$`, federations.Get, auth.PrivLevelFederation, Authenticated, nil},
 		{1.1, http.MethodPost, `federations/{id}/deliveryservices?(\.json)?$`, federations.PostDSes, auth.PrivLevelAdmin, Authenticated, nil},
 		{1.1, http.MethodGet, `federations/{id}/deliveryservices?(\.json)?$`, api.ReadHandler(&federations.TOFedDSes{}), auth.PrivLevelReadOnly, Authenticated, nil},
-		{1.1, http.MethodDelete, `federations/{id}/deliveryservices?(\.json)?$`, api.DeleteHandler(&federations.TOFedDSes{}), auth.PrivLevelAdmin, Authenticated, nil},
+		{1.1, http.MethodDelete, `federations/{id}/deliveryservices/{dsID}/?(\.json)?$`, api.DeleteHandler(&federations.TOFedDSes{}), auth.PrivLevelAdmin, Authenticated, nil},
```

Comment:
Why remove the existing handler?

Reply:
?

Reply:
This removes the handler for `DELETE /api/1.x/federations/(ID}/deliveryservices`

Reply:
yeah it does from my initial commit but not master. my initial commit I was replacing the perl path of _federations/{id}/deliveryservices/{dsID}_  with _/api/1.x/federations/(ID}/deliveryservices_ but that was going to break the API so i backed it out and went back to the correct path

Reply:
oh... How did I even comment on that?